healthtools/scrapers/base_scraper.py

```diff
@@ -225,43 +229,41 @@ def archive_data(self, payload):
         '''
         Upload scraped data to AWS S3
         '''
-        data_key = DATA_DIR + self.data_key
-        data_archive_key = DATA_DIR + self.data_archive_key
         try:
             date = datetime.today().strftime("%Y%m%d")
             if AWS["s3_bucket"]:
                 old_etag = self.s3.get_object(
-                    Bucket=AWS["s3_bucket"], Key=data_key)["ETag"]
+                    Bucket=AWS["s3_bucket"], Key=self.data_key)["ETag"]
                 new_etag = hashlib.md5(payload.encode("utf-8")).hexdigest()
                 if eval(old_etag) != new_etag:
                     file_obj = StringIO(payload.encode("utf-8"))
                     self.s3.upload_fileobj(file_obj,
-                                           AWS["s3_bucket"], data_key)
+                                           AWS["s3_bucket"], self.data_key)
 
                     # archive historical data
                     self.s3.copy_object(Bucket=AWS["s3_bucket"],
-                                        CopySource="{}/".format(AWS["s3_bucket"]) + data_key,
-                                        Key=data_archive_key.format(date))
+                                        CopySource="{}/".format(AWS["s3_bucket"]) + self.data_key,
+                                        Key=self.data_archive_key.format(date))
                     print "[{0}] Archive: Data has been updated.".format(datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
                     return
                 else:
                     print "[{0}] Archive: Data scraped does not differ from archived data.".format(
                         datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
             else:
-                # check if it's test and append the correct path
-                if "test" in self.data_key:
-                    self.data_key = TEST_DIR + self.data_key
```

Comment:
@Gathondu Yip. That's my question - it looks like you've removed the test scenario.

Also, it looks like if we have `AWS["s3_bucket"]` set, we don't actually upload to a test directory. We should fix this.